tests/test_data/input_test_generate/vehicle_types.json

```diff
@@ -0,0 +1,48 @@
+{
+    "sprinter_small": {
+        "name": "Sprinter (small)",
+        "capacity": 38,
+        "mileage": 25,
+        "charging_curve": [[0, 11], [0.8, 11], [1, 11]],
+        "min_charging_power": 0.2,
+        "v2g": false
+    },
+    "sprinter": {
+        "name": "Sprinter",
+        "capacity": 76,
+        "mileage": 40,
+        "charging_curve": [[0, 11], [0.8, 11], [1, 11]],
+        "min_charging_power": 0.2,
+        "v2g": true
+    },
+    "golf": {
+        "name": "E-Golf",
+        "capacity": 50,
+        "mileage": 16,
+        "charging_curve": [[0, 22], [0.8, 22], [1, 22]],
+        "min_charging_power": 0,
+        "v2g": true
+    },
+    "AB-OPP": {
+        "name": "articulated bus - opportunity charging",
+        "capacity": 50,
+        "charging_curve": [[0, 150], [0.8, 150], [1, 150]],
+        "min_charging_power": 0,
+        "v2g": false
+    },
+    "minimum_information": {
+        "name": "Displayed name",
+        "capacity": 0,
+        "mileage": 0,
+        "charging_curve": [[0,1],[1,0]]
+    },
+    "all_information": {
+        "name": "Displayed name",
+        "capacity": 0,
+        "mileage": 0,
+        "charging_curve": [[0,1],[1,0]],
+        "min_charging_power": 0,
+        "v2g": false,
+        "battery_efficiency": 0.95
+    }
```

Comment:
Are these three vehicle_types needed? I did not find these in the test_generate.

Reply:
true, i will clear it out